Makefile

```diff
@@ -0,0 +1,48 @@
+
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#
+# Supported Targets:
+#
+# fabric-unit-test:         runs unit tests
+# lint:                     runs linters
+# checks:                   runs code checks
+#
+
+CONTAINER_IDS = $(shell docker ps -a -q)
+DEV_IMAGES = $(shell docker images dev-* -q)
+export GO111MODULE=on
+
+depend: version
+	@scripts/dependencies.sh
+
+checks: version depend lint
+
+lint:
+	@scripts/check_lint.sh
+
+all: checks fabric-unit-test
+
+fabric-unit-test: export FABRIC_COMMAND=unit-test
```

Comment:
there are two targets with the same name here?
also - are we expecting another type of unit-test?

Reply:
done

Reply:
I mean should the command in line 30 be part of the target in line 31?

Reply:
I can't set env var within the same target I had to create another one with same name